src/routes.rs

```diff
@@ -28,6 +28,7 @@ pub fn handle_request(
     authorizor: &dyn auth::Authorizor,
     storage: &dyn storage::Storage,
     auth_proxy: bool,
+    fastly_authed: bool,
```

Comment:
We're starting to get this function a little [bool-ified](https://alexkondov.com/should-you-pass-boolean-to-functions/), do you think these booleans could live on the `Authorizor` trait?

Reply:
Will post a followup.